README.md

```diff
@@ -75,6 +75,13 @@ We welcome community contributions to this project.
 
 Please read our [Contributor Guide](CONTRIBUTING.md) for more information on how to get started.
 
+## Releasing
+
+We use the [cargo release](https://github.com/crate-ci/cargo-release) tool to manage changelogs, git tags and publishing crates.
+
+Each substantial pull request should have a changelog entry (see [keep a changelog](https://keepachangelog.com)) and never update
+a crate version directly.
```

Comment:
Thanks for adding this section!

There are two things that could be clarified further though:

Should we manually fill the changelog, or does `cargo-release` do it for us?

> never update a crate version directly

What does this mean?